components/StandardListItem/StandardListItem.jsx

```diff
@@ -0,0 +1,46 @@
+import {PropTypes, Component } from 'react'
+import React from 'react'
+
+require('./StandardListItemStyles.scss')
+ // showIcon: true -> render the icon
+ // showLabel: true -> render the label
+ // imgSrc: source for the icon
+ // labelText: The text for the label
+ // placeIcon: defines the position of the icon. Either: top | left | right.  Default to top
+
+class StandardListItem extends Component {
+  constructor(props) {
+    super(props)
+  }
+  render() {
+    const classes = 'StandardListItem ' + this.props.placeIcon
+    let label
+    let icon
+
+    if (this.props.showLabel){
+      label = <p className="label">{this.props.labelText}</p>
+    }
+
+    if (this.props.showIcon){
+      icon = <img className="icon" src={this.props.imgSrc}/>
+    }
+
+    return (<div className={classes}>{label}{icon}</div>)
+  }
+}
+
+StandardListItem.propTypes = {
```

Comment:
If any of these propTypes are required for the component, there should be `.isRequired` added to it, e.g. `showIcon: PropTypes.bool.isRequired`